test/3x/torch/quantization/test_pt2e_quant.py

```diff
@@ -283,3 +282,56 @@ def test_mixed_fp16_and_int8(self, force_not_import_ipex):
         opt_model = torch.compile(converted_model)
         out = opt_model(*example_inputs)
         assert out is not None
+
+    @pytest.mark.skipif(not GT_OR_EQUAL_TORCH_VERSION_2_5, reason="Requires torch>=2.5")
+    @pytest.mark.parametrize("half_precision_dtype", ["fp16", "bf16"])
+    @pytest.mark.parametrize("op_name", ["conv1", "fc1"])

```

Comment:
Do we support op_type level autotune?

Reply:
Yes, extended UTs to test the op_type as well at [c180739](https://github.com/intel/neural-compressor/pull/2110/commits/c1807391b013361c38f2a4b75f625cdbbe2d0836).